packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/web_camera_control_position.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Identifiers used to specify the placement of controls on the map.
+// Controls are positioned relative to other controls in the same layout position.
+// Controls that are added first are positioned closer to the edge of the map.
+// Usage of "logical values"
+// (see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values)
+// is recommended in order to be able to automatically support both
+// left-to-right (LTR) and right-to-left (RTL) layout contexts.
+
+/*
+Logical values in LTR:
+
++----------------+ 
+| BSIS BSIC BSIE | 
+| ISBS      IEBS | 
+|                | 
+| ISBC      IEBC | 
+|                | 
+| ISBE      IEBE | 
+| BEIS BEIC BEIE | 
++----------------+
+
+Logical values in RTL:
+
++----------------+ 
+| BSIE BSIC BSIS | 
+| IEBS      ISBS | 
+|                | 
+| IEBC      ISBC | 
+|                | 
+| IEBE      ISBE | 
+| BEIE BEIC BEIS | 
++----------------+
+
+Legacy values:
+
++----------------+ 
+| TL    TC    TR | 
+| LT          RT | 
+|                | 
+| LC          RC | 
+|                | 
+| LB          RB | 
+| BL    BC    BR | 
++----------------+
+*/
```

Comment:
What's the purpose of duplicating this from the link below? It's not part of the doc comment, and the plugin will never be responsible for actually implementing this behavior, so it's not clear to me who the audience for this comment is.

Reply:
Good point. I removed the duplicated info.